connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -284,16 +288,18 @@ private void doCommitSync(Map<TopicPartition, OffsetAndMetadata> offsets, int se
      * the write commit.
      **/
     private void doCommit(Map<TopicPartition, OffsetAndMetadata> offsets, boolean closing, final int seqno) {
-        log.info("{} Committing offsets", this);
+        log.info("{} Committing offsets: {}", this, offsets);
         if (closing) {
             doCommitSync(offsets, seqno);
         } else {
             OffsetCommitCallback cb = new OffsetCommitCallback() {
                 @Override
                 public void onComplete(Map<TopicPartition, OffsetAndMetadata> offsets, Exception error) {
+                    log.debug("{} Complete async offset commit callback for generation {}: {}", this, seqno, offsets);
                     onCommitCompleted(error, seqno, offsets);
                 }
             };
+            log.debug("{} Begin async offset commit for generation {}: {}", this, seqno, offsets);
```

Comment:
I wonder if this message and the one in `doCommitSync` is overkill. Maybe we could change the first log message in `doCommit` to include whether it is async or sync. For example?
```java
boolean syncCommit = closing;
log.info("{} Committing {} offsets: {}", this, isSyncCommit? "sync" : "async", offsets);
```